holmes_operator/handlers/healthcheck.py

```
@@ -0,0 +1,258 @@
+        logger.error(
+            f"Failed to execute HealthCheck {namespace}/{name}: {e}",
+            exc_info=True,
+            extra={"check_name": name, "namespace": namespace, "error": str(e)},
+        )
+
+        # Update status to Failed
+        await set_healthcheck_failed(
+            api=context.k8s_api,
+            name=name,
+            namespace=namespace,
+            message=f"Operator error: {str(e)}",
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)